astroplan/plots/mplstyles.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.visualization import astropy_mpl_style
+
+__all__ = ['light_style_sheet', 'dark_style_sheet',
+           'available_style_sheets']
+
+available_styles = ["light_style_sheet", "dark_style_sheet"]
+
+def available_style_sheets():
```

Comment:
I suggest this function be removed, users should just use `available_styles` directly.
In Sphinx if you put a comment below a module-level variable, it should show up in the API docs:
```python
available_styles = ["light_style_sheet", "dark_style_sheet"]
"""Matplotlib style sheets available within astroplan."""
```
@eteq – But I think this is still broken for Astropy and all affiliated packages?

Still, even if that is broken at the moment, I prefer to have the better API, i.e remove this function.